spec/requests/statsd_middleware_spec.rb

```diff
@@ -1,108 +0,0 @@
-# frozen_string_literal: true
-
-require 'rails_helper'
```

Comment:
Do not delete this file. We need to refactor so that it uses an [anonymous controller](https://rspec.info/features/6-0/rspec-rails/controller-specs/anonymous-controller/)